bin/registerServerTopology.sh

```diff
@@ -247,8 +247,8 @@ EOF
 #========= begin of get_api_password()
 function get_api_password() {
   stty -echo
-  echo "$1 : Please enter your password:"
-  read -r API_TARGET_PASSWORD
+  read -r PASSWORD
+  echo $PASSWORD
   stty echo
```

Comment:
I would have put the result in a variable instead of using stty

Reply:
Shouldn't you check that at least one character is given ?

Reply:
IMHO no, its credentials are false (empty password included), the API will return a 401.

Using stty will hide the password so it couldn't be retrieve with a grep or an history

Reply:
but you can avoid the use of the echo and the stty, as the read result is saved and returned : 
https://github.com/centreon/centreon/pull/9604/files#diff-7dbe79a5ef0f7804f671663ed96391f693dc5a5b6e58315eeb55faee3303c5a8R439